testing/simapp/app.go

```diff
@@ -333,24 +339,31 @@ func NewSimApp(
 	transferModule := transfer.NewAppModule(app.TransferKeeper)
 	transferIBCModule := transfer.NewIBCModule(app.TransferKeeper)
 
+	// NOTE: the IBC mock keeper and application module is used only for testing core IBC. Do
+	// note replicate if you do not need to test core IBC or light clients.
```

Comment:
Same typo as above! Copy pasta 🍝 

```suggestion
	// not replicate if you do not need to test core IBC or light clients.
```